specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+            "get": {
+                "tags": [
+                    "Operations"
+                ],
+                "operationId": "Operations_List",
+                "description": "Lists all of the available Azure attestation operations.",
+                "x-ms-examples": {
+                    "Operations_List": {
+                        "$ref": "./examples/Operations_List.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "responses": {
+                    "200": {
+                        "description": "List all the operations.",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    },
+                    "default": {
+                        "description": "Attestation service error result",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResult"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Attestation/attestationProviders/{providerName}": {
+            "get": {
+                "tags": [
+                    "AttestationProvider"
+                ],
+                "operationId": "AttestationProviders_Get",
+                "description": "Get the status of Attestation Provider.",
+                "x-ms-examples": {
+                    "AttestationProviders_Get": {
+                        "$ref": "./examples/Get_AttestationProvider.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "name": "providerName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9-]{3,24}$",
+                        "description": "Name of the attestation service"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Get the status of attestation Provider",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "404": {
```

Comment:
There is no need to model specific error codes. Those are handled by "default"

Reply:
removed 
                    "404": {
                        "description": "Attestation provider not found"
                    },